ui/lib/kmip/addon/components/header-credentials.js

```diff
@@ -1,11 +1,13 @@
-import Component from '@ember/component';
+import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
-import layout from '../templates/components/header-credentials';
 
-export default Component.extend({
-  layout,
-  tagName: '',
-  secretMountPath: service(),
-  scope: null,
-  role: null,
-});
+export default class HeaderCredentialsComponent extends Component {
+  @service secretMountPath;
```

Comment:
Not sure why this component needs this service... it isn't being invoked anywhere that I can tell. Do you have any insight there?

Reply:
Good catch. I didn't even notice that. I'll remove and then test to make sure.